projects/observability/src/pages/apis/api-detail/api-detail-breadcrumb.resolver.test.ts

```diff
@@ -108,7 +130,7 @@ describe('Api detail breadcrumb resolver', () => {
         entityType: ObservabilityEntityType.Api,
         id: 'test-id'
       }),
-      { cacheability: GraphQlRequestCacheability.NotCacheable }
```

Comment:
Will have to keep an eye on this change. Hopefully it doesn't break any existing case

Reply:
you could change the default from cacheable to undefined too - use the default options.